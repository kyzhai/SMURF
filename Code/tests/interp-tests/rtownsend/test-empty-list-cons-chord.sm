/* Testing how empty lists are treated by interpreter */

main = [] : [(2,0)$4]
