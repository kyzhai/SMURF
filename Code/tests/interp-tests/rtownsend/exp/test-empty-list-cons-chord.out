
Timing Resolution set to 4 PPQ

Instrument set to 48 on channel 0

Track 0:
 tick 0, channel 1: program change 48
 tick 0, channel 1: note D2 on velocity: 90
 tick 4, channel 1: note D2 on velocity: 0
 tick 4, end of track
Program: 
	SMain: [] : [ (2, 0)$4]
Global Scope: 
	ID: main  :: Chord
	ID: print x :: a -> a
	ID: random  :: Int
	ID: head h : t :: [a] -> a
	ID: tail h : t :: [a] -> [a]

java -jar ./Lib/CSV2MIDI.jar a.csv a.midi
===== Program Successfully Finished =====
===== Result Writen to a.midi =====
